app/frontend/wallet/cardano-trezor-crypto-provider.js

```diff
@@ -122,8 +120,13 @@ const CardanoTrezorCryptoProvider = (ADALITE_CONFIG, walletState) => {
     throw new Error('Unsupported operation!')
   }
 
+  function getWalletDerivationScheme() {
```

Comment:
I would just call it `getDerivationScheme` - the crypto provider does not have a notion of "wallet"